python/kfserving/README.md

```diff
@@ -45,6 +45,11 @@ KFServing supports the following storage providers:
     * Absolute path: `/absolute/path` or `file:///absolute/path`
     * Relative path: `relative/path` or `file://relative/path`
     * For local filesystem, we recommended to use relative path without any prefix.
+* Persistent Volume Claims with the format "pvc://{$pvcname}/[path]".
+    * The `pvcname` is the name of Persistent Volume Claims that storages models.
```

Comment:
s/Claims/Claim
s/storages/stores

Reply:
On second, thought:
The `pvcname` is the name of the Persistent Volume Claim that contains the model.